public/components/test.js

```diff
@@ -1,5 +1,5 @@
 import './game/details/details_test';
-//import './navigation/navigation_test';
+// import './navigation/navigation_test';
```

Comment:
Yeah I can see that. Let me know if you prefer for it to be removed or left in but commented out. When Mike uncommented those tests they didn't pass. We didn't get to that as a part of our upgrade work, honestly not even realizing they were there until later today; but, it was already in the file and commented out.